build.gradle

```diff
@@ -11,6 +20,7 @@ plugins {
 }
 
 apply from: 'gradle/scripts/yaml.gradle'
+apply plugin: 'org.junit.platform.gradle.plugin'
```

Comment:
Ugh, but at least the JUnit team is aware of the issue: junit-team/junit5#768.  However, it appears to have been pushed back to 5.1 as of 14 hours ago. 😞 

Note that there's a [workaround](https://github.com/junit-team/junit5/issues/768#issuecomment-330078905) listed in the issue, but I'm not sure if it's worth introducing that just to keep `build.gradle` clean.

Nit: Pasting the `buildscript` configuration from the JUnit docs introduced mixed tabs/spaces into this file.